graf2d/gpad/v7/inc/ROOT/TPadUserCoord.hxx

```diff
@@ -43,7 +43,7 @@ protected:
    TPadUserCoordBase() = default;
 
 public:
-   virtual ~TPadUserCoordBase();
+   virtual ~TPadUserCoordBase() {}
```

Comment:
These function should really go into TPadUserCoordBase.cxx (which doesn't exist yet). Same for `~TPadLinearUserCoord()`. (Allows the linker to "pin the vtable" which otherwise gets emitted for every translation unit that includes this header...)

Can you change that in a subsequent PR?